<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainMenu</class>
 <widget class="QWidget" name="MainMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>790</width>
    <height>689</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(43, 43, 43);</string>
  </property>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>580</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
color:white;
font: &quot;Arial&quot;;
</string>
   </property>
   <property name="text">
    <string>Подключено!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_mm">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>120</y>
     <width>141</width>
     <height>40</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>150</width>
     <height>40</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{

background-color:qlineargradient(spread:pad, x1:0.494, y1:0.846136, x2:0.494382, y2:1, stop:0 rgba(69, 69, 69, 255), stop:1 rgba(149, 149, 149, 255));
color:white;
font: &quot;Arial&quot;;
border-radius: 2px;
}
QPushButton::pressed{

background-color:qlineargradient(spread:pad, x1:0.426, y1:0.800682, x2:0.427, y2:1, stop:0 rgba(37, 37, 37, 255), stop:1 rgba(87, 87, 87, 255));
color:white;
font: &quot;Arial&quot;;
border-radius: 2px;
};
</string>
   </property>
   <property name="text">
    <string>Быстрая игра</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_exit">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>420</y>
     <width>141</width>
     <height>40</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>150</width>
     <height>40</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{

background-color:qlineargradient(spread:pad, x1:0.494, y1:0.846136, x2:0.494382, y2:1, stop:0 rgba(69, 69, 69, 255), stop:1 rgba(149, 149, 149, 255));
color:white;
font: &quot;Arial&quot;;
border-radius: 2px;
}
QPushButton::pressed{

background-color:qlineargradient(spread:pad, x1:0.426, y1:0.800682, x2:0.427, y2:1, stop:0 rgba(37, 37, 37, 255), stop:1 rgba(87, 87, 87, 255));
color:white;
font: &quot;Arial&quot;;
border-radius: 2px;
};
</string>
   </property>
   <property name="text">
    <string>Выйти</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_connect_group">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>240</y>
     <width>141</width>
     <height>40</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>150</width>
     <height>40</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{

background-color:qlineargradient(spread:pad, x1:0.494, y1:0.846136, x2:0.494382, y2:1, stop:0 rgba(69, 69, 69, 255), stop:1 rgba(149, 149, 149, 255));
color:white;
font: &quot;Arial&quot;;
border-radius: 2px;
}
QPushButton::pressed{

background-color:qlineargradient(spread:pad, x1:0.426, y1:0.800682, x2:0.427, y2:1, stop:0 rgba(37, 37, 37, 255), stop:1 rgba(87, 87, 87, 255));
color:white;
font: &quot;Arial&quot;;
border-radius: 2px;
};
</string>
   </property>
   <property name="text">
    <string>Присоединиться </string>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_profile">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>300</y>
     <width>141</width>
     <height>40</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>150</width>
     <height>40</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{

background-color:qlineargradient(spread:pad, x1:0.494, y1:0.846136, x2:0.494382, y2:1, stop:0 rgba(69, 69, 69, 255), stop:1 rgba(149, 149, 149, 255));
color:white;
font: &quot;Arial&quot;;
border-radius: 2px;
}
QPushButton::pressed{

background-color:qlineargradient(spread:pad, x1:0.426, y1:0.800682, x2:0.427, y2:1, stop:0 rgba(37, 37, 37, 255), stop:1 rgba(87, 87, 87, 255));
color:white;
font: &quot;Arial&quot;;
border-radius: 2px;
};
</string>
   </property>
   <property name="text">
    <string>Профиль</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_create_group">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>180</y>
     <width>141</width>
     <height>40</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>150</width>
     <height>40</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{

background-color:qlineargradient(spread:pad, x1:0.494, y1:0.846136, x2:0.494382, y2:1, stop:0 rgba(69, 69, 69, 255), stop:1 rgba(149, 149, 149, 255));
color:white;
font: &quot;Arial&quot;;
border-radius: 2px;
}
QPushButton::pressed{

background-color:qlineargradient(spread:pad, x1:0.426, y1:0.800682, x2:0.427, y2:1, stop:0 rgba(37, 37, 37, 255), stop:1 rgba(87, 87, 87, 255));
color:white;
font: &quot;Arial&quot;;
border-radius: 2px;
};
</string>
   </property>
   <property name="text">
    <string>Создать комнату</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>580</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
color:white;
font: &quot;Arial&quot;;
</string>
   </property>
   <property name="text">
    <string>Игроков в сети:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_settings">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>360</y>
     <width>141</width>
     <height>40</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>150</width>
     <height>40</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{

background-color:qlineargradient(spread:pad, x1:0.494, y1:0.846136, x2:0.494382, y2:1, stop:0 rgba(69, 69, 69, 255), stop:1 rgba(149, 149, 149, 255));
color:white;
font: &quot;Arial&quot;;
border-radius: 2px;
}
QPushButton::pressed{

background-color:qlineargradient(spread:pad, x1:0.426, y1:0.800682, x2:0.427, y2:1, stop:0 rgba(37, 37, 37, 255), stop:1 rgba(87, 87, 87, 255));
color:white;
font: &quot;Arial&quot;;
border-radius: 2px;
};
</string>
   </property>
   <property name="text">
    <string>Настройки</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_num_players">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>580</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
color:white;
font: &quot;Arial&quot;;
</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;нет &lt;/p&gt;&lt;p&gt;связи с сервером&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>253</y>
     <width>261</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
color:white;
font: &quot;Arial&quot;;
</string>
   </property>
   <property name="text">
    <string>На сервере нет ни одной группы! Создайте свою!</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
